tools/depends/target/ffmpeg/CMakeLists.txt

```diff
@@ -64,9 +64,7 @@ elseif(CORE_SYSTEM_NAME STREQUAL android)
   endif()
   list(APPEND ffmpeg_conf --target-os=linux --extra-libs=-liconv --disable-linux-perf)
 elseif(CORE_SYSTEM_NAME STREQUAL darwin_embedded)
-  if(NOT CPU MATCHES arm64)
-    list(APPEND ffmpeg_conf --cpu=cortex-a8)
-  else()
+  if(CPU MATCHES arm64)
```

Comment:
Is there any case where CPU isn't arm64?

Reply:
I don't believe so, but if there is then I should revert this?